### PR TITLE
Cleanup & Adjustment to support static site generation

### PR DESCRIPTION
**Pull request by N3XT14**

### CODE REVIEW

**Comment by N3XT14** [comment]
@lucifercr07 Can you review this and let me know if something else needs to be done.

File: README.md [hunk 1]
```diff
@@ -118,6 +118,28 @@ To create a production build:
 npm run build
 ```
 
+## Creating a Static Production Build
+
+To generate a static production build of your Next.js application, follow these steps:
```
Comment (lucifercr07) [inline README.md:123]: We'd also need to move `export const SHELL_COMMAND_URL = 'http://localhost:8080/shell/exec';` as part of `.env` for this, otherwise won't be configurable.

File: src/components/Search/command.tsx [hunk 2]
```diff
@@ -1,14 +1,23 @@
-import { useState } from 'react';
 import { Clipboard } from 'lucide-react';
 import { DiceCmdMeta } from '@/data/command';
 
-export default function CommandPage({ title, syntax, body, url }: DiceCmdMeta) {
```
Comment (KaviiSuri) [inline src/components/Search/command.tsx:5]: this seems unrelated to static site generation, and it breaks the behaviour of "Copied" showing up is that expected?

File: src/shared/components/Button.tsx [hunk 3]
```diff
@@ -5,13 +5,25 @@ import React from 'react';
...
+  variant?: 'default' | 'outline'; // Added default variant
 }
 
-export function Button({ children, className }: ButtonProps) {
+export function Button({
+  children,
+  className = '',
+  variant = 'default',
+}: ButtonProps) {
+  const baseClasses =
```
Comment (KaviiSuri) [inline src/shared/components/Button.tsx:16]: doesn't seem like this should be inside the function component, would re-define on every render.

**Review by KaviiSuri — APPROVED** [review]
LGTM

**Review by lucifercr07 — APPROVED** [review]
